cpython/patches/testing.patch

```diff
@@ -82,3 +82,30 @@ index 1fc4de11e1..e6c91707ae 100644
  import time
  import unittest
  import unittest.mock as mock
+diff --git a/Lib/test/test_gzip.py b/Lib/test/test_gzip.py
+--- a/Lib/test/test_gzip.py
++++ b/Lib/test/test_gzip.py
+@@ -13,6 +13,7 @@
+ from test import support
+ from test.support import _4G, bigmemtest
+ from test.support.script_helper import assert_python_ok
++import unittest
+ 
+ gzip = support.import_module('gzip')
+ 
+@@ -691,6 +692,7 @@
+     data = b'This is a simple test with gzip'
+ 
+     def test_decompress_stdin_stdout(self):
++        raise unittest.SkipTest('no subprocess')
```

Comment:
Are you sure about this one? I wouldn't expect the test below to use subprocess..

Reply:
Never mind I see you also included the original traceback in your comment https://github.com/iodide-project/pyodide/pull/619#discussion_r389400332